src/storage-provider-local.ts

```diff
@@ -8,16 +7,16 @@ export default class LocalStorageProvider implements IStorageProvider {
         const repo = JSON.stringify(data);
         const key = `${this.prefix}:${name}`;
         try {
-            await AsyncStorage.setItem(key, repo);
+            window.localStorage.setItem(key, repo);
         } catch (ex) {
             console.error(ex);
         }
     }
 
-    public async get(name: string) {
+    public get(name: string) {
```

Comment:
Even though we do not need `async` marker on this method i think I would prefer to have it, to make sure we remember to await the get method also, sense this will be the default implementation. I can easily see myself (or someone else) forgetting the "await" in the calling code, if this method is not marked as async in the default implementation. This will lead to bugs, only visible when using a custom store.

Reply:
Evening! Not problem to get rid of async to but U've defined interface which is Promise based :) 
```typescript
// src/storage-provider.ts
export default interface IStorageProvider {
    save: (name: string, data: any) => Promise<void>;
    get: (name: string) => Promise<any>;
}
```

Reply:
good point :D